### ISSUE TITLE
IS22 Intern Standup

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/internship#238]
### Overview
We need to keep track of roll call and discussion items during the intern daily standup

### Action Items

- [x] Copy roll call into a comment
- [x] Mark attendance
- [x] Take relevant notes
- [ ] Update relevant WIKI page (See resources)

### Resources/Instructions
- Find WIKI page for your role [Internship Wiki, Join the team page](https://github.com/hackforla/internship/wiki/Joining-the-Team)

```

- Sakari: 
- Clayton
- Mateo
- Tung
- Riya
- Lily
- Charity
- Brian
- Beckett
- Asher: often has water polo practice

```

**Comment by ExperimentsInHonesty** [comment]
Sakari - trying to find an GHA issue
Clayton - blocker is where to change code
Mateo - going to take on a new issue
Tung - setting up dev environment 
Riya - wont run local host
Lily -

**Comment by ExperimentsInHonesty** [comment]
Clayton - put in pr but is not sure it went through
Tung - put in pull request
Brian - 90% done
beckett - done with two issues
Riya - finished with second issue
Charity - good progress
Lily - access to google drive
Sakari - GitHub action

**Comment by Rabia2219** [comment]
Monday June 27th

Clayton - finished 1 issue, reviewed 1 PR, working on fixing bugs in code
Tung - finished good 2nd issue, did 1 or 2 PR reviews, wants help on choosing a guide
Brian - good progress (will work on BallotNav)
Beckett - reviewed 1 PR, made an issue and worked on finding on a guide
Charity - good progress (needs to make a template in FigJam)
Lily - finished 1st issue, reviewed 2 PRs
Asher - figured out his blockers and can start working issues

**Comment by ExperimentsInHonesty** [comment]
Tung - waiting for 2nd pr review
Charity - figjam changes
Brian - working on making issues
Lilly - waiting on pr reviews
Clayton - working on small issue
Beckett - waiting on pr review on small issue
Lilly - waiting for 2 reviews (one reviewer assigned)

**Comment by ExperimentsInHonesty** [comment]
missing response from some of the students
https://docs.google.com/forms/d/1_2lrwm5cWpoew_xs7w5BCwu0bsho5gNJRDGYCI8o6co/edit#responses

**Comment by ExperimentsInHonesty** [comment]
Bonnie - don't forget to ask about linkedin

**Comment by Rabia2219** [comment]
@ExperimentsInHonesty add link to wiki in the resources

**Comment by ExperimentsInHonesty** [comment]
- Sakari: out June 27th to July 1st
- Clayton
- Mateo
- Tung
- Riya
- Lily
- Charity
- Brian
- Beckett - waiting for review, working on github actions
- Asher - not here

**Comment by ExperimentsInHonesty** [comment]
12pm sat with Charity, send invite
2pm sunday with Brian

**Comment by ExperimentsInHonesty** [comment]
- Sakari: got pr merged, have a new issue, worked on docker guide,   blockers are docker guides issue, finished react portion 
- Clayton: got merged and took up a medium issue and will meet with mentor tonight
- Mateo - not here
- Tung: crashing issues medium
- Riya: has not picked up a guide yet
- Lily: reset working environment
- Charity: having internet issues today
- Brian: will work on hfla issue
- Beckett: starting a medium issue and will finish following 
- Asher: not here

**Comment by ExperimentsInHonesty** [comment]
- Sakari: holding on issue until team meeting
- Clayton: met with mentor and no blockers
- Mateo: not
- Tung: fixed computer and is going to make a guide for how to test GHAs
- Riya: PR issue will address during meeting - will help her pick a guide
- Lily: got unstuck from the too large medium issue and is now on a new issue
- Charity: flow charts for migration project, reconciliation process of templates
- Brian: worksheet for hack for LA website migration
- Beckett: Decision Record review blocker, Sakari will review
- Asher: not here

**Comment by ExperimentsInHonesty** [comment]
ask interns if they can do Friday the 29th at 11am for exit interviews or if they can do later

**Comment by Rabia2219** [comment]
Reminder:
ask interns if they can do Friday the 29th at 11am for exit interviews or if they can do later

**Comment by ExperimentsInHonesty** [comment]
- Sakari: will go to office hours on the issue he has a blocker
- Clayton: will go to office hours tonight
- Mateo
- Tung: will go to office hours
- Riya: can't come today on a flight
- Lily: waiting for review
- Charity: 
- Brian - hfla website, will add roster info
- Beckett: 2 prs, ava helped with guides outreach, waiting on a test to run, blocker able to test on ghpages repo, will discuss during office hours.
- Asher

**Comment by ExperimentsInHonesty** [comment]
- Sakari: reviewed a few pull requests, and took on a CSS issue, blocker is spent a bunch of time with cleaning up files that get added to all your branches.
- Clayton:  front end, medium
- Mateo:  no here
- Tung:  working on the issue, there is no template, she will make one.
- Riya: issue got merged, busy this week with 
- Lily, will need to figure out location issue
- Charity:  working on video, limited by 
- Brian: worked on sheet
- Beckett: reviewed prs 
- Asher: not here

**Comment by Rabia2219** [comment]
@ExperimentsInHonesty
reminder: meet with Tung after standup https://github.com/hackforla/website/issues/2816

**Comment by ExperimentsInHonesty** [comment]
Ask them to take [linkedin survey](https://docs.google.com/forms/d/e/1FAIpQLSdNByzgUK19HlWSmdyuegaMwHrVGTe4d1Phq0pJZ9DCtjs-sw/viewform)
Ask them to update [linkedin issues](https://github.com/hackforla/internship/issues?q=is%3Aissue+is%3Aopen+label%3A%22role%3A+intern%22+label%3A%22p-feature%3A+Linkedin+profile%22+)

**Comment by ExperimentsInHonesty** [comment]
Who can confirm Friday 3-4 pm for exit interviews
we will schedule with the Thursday people individually early next week

**Comment by ExperimentsInHonesty** [comment]
- Sakari:  still working on his isssue, crushing errors, pr reviews, can do friday
- Clayton: working on the issue, no blockers, friday
- Mateo: not here
- Tung: not here
- Riya: working on issues, no blockers, can't get access to dashboard, can do friday
- Lily: working on issues and prs, can do thursday (as early as you want (because its daytime in france) no later than 12pm.
- Charity: video and progress on issues, friday ok
- Brian: compilation spreadsheets, no blocker
- Beckett: making progress,  can do friday
- Asher: often has water polo practice

**Comment by Rabia2219** [comment]
@ExperimentsInHonesty 
Putting the updated/fixed [TWE: End of Week Mentor Survey](https://docs.google.com/forms/d/e/1FAIpQLSe1iAi5xpOW5EXyAAKHbjdDUt_scR02Oy0bkiligSyuQEi1dw/viewform) here if you want to ask the mentee students to fill it in...

**Comment by ExperimentsInHonesty** [comment]
- Sakari: b-day: refactor skills 
- Clayton: testing your GHA issue
- Mateo: testing issue
- Tung: 
- Riya: has not read new issue
- Lily: reviewing prs and looking for new issue
- Charity: trying to find a free video editor 
- Brian:  hidden drive issue
- Beckett: reviewing prs and working on guides - mentioned this https://github.com/nektos/act
- Asher: waiting for issue to be merged

**Comment by christine-el** [comment]
@ExperimentsInHonesty @epierotti3 The following interns have not yet responded "yes" to the calendar invite for their exit interview.  Please remind them to respond.  I will also follow up on Slack this evening if I don't hear from them.

- Lily-Interview scheduled for Thursday from 9:30-10:30 am PST
- Riya-Interview scheduled for Friday from 3-4 pm PST
- Clayton-Interview scheduled for Friday from 3-4 pm PST
- Mateo- Interview schedule for Friday from 3-4 pm PST (I will reach out via email to see if this time works for him.)
- Brian-Interview scheduled for Friday from 3-4 pm PST

**Comment by ExperimentsInHonesty** [comment]
- Sakari: finished issue and looking for new one, 
- Clayton: working with running GHAs locally
- Mateo: not here
- Tung: have a question X
- Riya: new issue, have a question X
- Lily: not here
- Charity: making video
- Brian: double check worksheet
- Beckett: PRs and waiting for review (will ask about it in 
- Asher: often has water polo practice

**Comment by ExperimentsInHonesty** [comment]
Book time with Bonnie if your linkedin experience section still needs work

**Comment by epierotti3** [comment]
@ExperimentsInHonesty [intern off-boarding form](https://docs.google.com/forms/d/160XMPcu_b2vDCKFYSUjwC_qC2ZdLBbGcgrfXZoFYO34/viewform) to send out during standup

**Comment by ExperimentsInHonesty** [comment]
lilly and charity have filled out the form

**Comment by epierotti3** [comment]
Ask Lily at standup on Thursday 7/28 about scheduling her exit interview

Form update: all but Sakari and Mateo have filled out the off-boarding form

**Comment by ExperimentsInHonesty** [comment]
- Sakari: 
- Clayton
- Mateo
- Tung: issue with no labels https://github.com/hackforla/website/pull/3428, https://github.com/hackforla/website/pull/3024
- Riya
- Lily
- Charity
- Brian
- Beckett
- Asher: often has water polo practice
